src/runner_manager_type.py

```diff
@@ -16,6 +17,17 @@
 from runner_type import GithubPath
 
 
+class FlushMode(Enum):
+    """Strategy for flushing runners."""
+
+    FLUSH_IDLE = 0
+    """Flush only idle runners."""
```

Comment:
Shouldn't this go into the docstring? I'm guessing that our linting will complain that these attributes are not explained in the class docstring

Reply:
Updated to the class docstring.